hwdb/60-sensor.hwdb

```diff
@@ -363,6 +363,10 @@ sensor:modalias:acpi:*BOSC0200*:dmi:*:svnLENOVO*:pn80U1:*
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnLINX*:pnLINX1010B:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, -1
 
+# Linx 12V64
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnLINX*:pnLINX12V64:*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
```

Comment:
please merge with next entry, which is the same in all ways, except for V vs. X

Reply:
I merged the two entries and added a wildcard in my latest commit.